test/helpers/envtest.go

```diff
@@ -59,9 +80,15 @@ func init() {
 	utilruntime.Must(clusterv1.AddToScheme(scheme.Scheme))
 	utilruntime.Must(bootstrapv1.AddToScheme(scheme.Scheme))
 	utilruntime.Must(expv1.AddToScheme(scheme.Scheme))
-	utilruntime.Must(controlplanev1.AddToScheme(scheme.Scheme))
+	// add crs to envtest
```

Comment:
nit:
I would suggest removing this comment (why this and not the other?)
Also, I think we should remove empty lines between AddToScheme calls